### PR TITLE
Add new workflow for public-workflows

### PR DESCRIPTION
**Pull request by cgundy**

### CODE REVIEW

File: .github/workflows/check_signed_new.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Install Dependencies
+        run: pip install -q -r requirements.txt
+        shell: bash
+      - name: Check CLA issue
+        run: |
+          export PYTHONPATH="$PWD/reusable_workflows/"
+          python reusable_workflows/check_cla/check_cla_pr.py
```
Comment (levsha) [inline .github/workflows/check_signed_new.yaml:27]: The path seems to be wrong.

**Review by levsha — COMMENTED** [review]
Was it tested anyhow?

File: .github/workflows/check_signed_new.yaml (hunk 1 above)
Comment (cgundy) [reply .github/workflows/check_signed_new.yaml:27]: It is correct on this branch https://github.com/dfinity/public-workflows/tree/remove-action-files/reusable_workflows/check_cla

**Comment by cgundy** [comment]
> Was it tested anyhow?

I haven't tested it yet, but I added this line:

```
if: ${{ github.event.label.name == 'test' }}
```
So I can test it independently.

**Review by levsha — APPROVED** [review]

**Pull request merged**